certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -302,26 +302,36 @@
 		"/chat-api/v2/messages/": {
 			"post": {
 				"responses": {
-					"default": {
-						"description": "default",
-						"schema": {}
+					"202": {
```

Comment:
@ridlees Indentations - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@ridlees Do you have any update on this?

Reply:
@sriyen-msft Hello, sorry, just pushed a fix. (tested locally and via paccon